tests/system/action/user/test_set_password_self.py

```diff
@@ -24,7 +23,7 @@ def test_update_wrong_password(self) -> None:
         old_hash = self.auth.hash("old")
```

Comment:
The `update` in the tests name is not fitting.

Reply:
Renamed.